django_extensions/db/fields/encrypted.py

```diff
@@ -6,6 +6,7 @@
 from django.conf import settings
 from django.core.exceptions import ImproperlyConfigured
 from django.db import models
+import logging
```

Comment:
unused import ?

Reply:
Good catch, was using this to debug. Have some more changes coming in as well. Will push them together